src/VisualStudio/Core/Def/ProjectSystem/FileChangeWatcher.cs

```diff
@@ -44,10 +44,10 @@ public FileChangeWatcher(
     {
         _fileChangeService = fileChangeService;
 
-        // 📝 Empirical testing during high activity (e.g. solution close) showed strong batching performance even
-        // though the batching delay is 0.
+        // 📝 Empirical testing during high activity (e.g. solution open/close) showed strong batching performance
+        // with batching delay of 500 ms.
```

Comment:
I have no particular opinion what this number should be, but do we have some way to actually differentiate between zero and 500 ms for which is better here? If both numbers were "showing strong batching performance" how did we come up with this change?

Reply:
I did not! Maybe we should add that data as a comment to the code.

Reply:
Thanks for the extra comment!